phpstan-baseline.neon

```diff
@@ -625,6 +625,36 @@ parameters:
 			count: 1
 			path: src/lib/eZ/FieldType/RichText/RichTextStorage.php
 
+		-
+			message: "#^Offset 1 does not exist on array\\{0\\?\\: string, 1\\?\\: ''\\|'ezremote\\://', 2\\?\\: string, 3\\?\\: string\\}\\.$#"
+			count: 1
+			path: src/lib/eZ/FieldType/RichText/RichTextStorage.php
```

Comment:
Note: allowing this since this is baseline alignment after an update and fixing this requires some changes that would take significant amount of time (impl + QA)

Reply:
yeah, and the same thing was done in 4.6 : https://github.com/ibexa/fieldtype-richtext/commit/560eecf461c7bbba36dd1d1df9fb8bcf338ad34f